server/src/main/java/org/bonitasoft/console/common/server/page/CustomPageService.java

```diff
@@ -307,11 +307,17 @@ public Properties getPageProperties(final APISession apiSession, final byte[] zi
         } else {
             properties = pageAPI.getPageProperties(zipContent, false);
             if (checkIfItAlreadyExists) {
+                final String pageName = properties.getProperty(NAME_PROPERTY);
                 try {
-                    final String pageName = properties.getProperty(NAME_PROPERTY);
                     pageAPI.getPageByNameAndProcessDefinitionId(pageName, processDefinitionId);
                     throw new AlreadyExistsException("A page with name " + pageName + " already exists for the process " + processDefinitionId);
                 } catch (final PageNotFoundException e) {
+                    try {
+                        pageAPI.getPageByName(pageName);
+                        throw new AlreadyExistsException("A page with name " + pageName + " already exists for the process " + processDefinitionId);
```

Comment:
I believe the message of the exception sshould be changed ("for the tenant" instead of "for the process")